src/frontendHighlighterApp/sass/app.scss

```
@@ -224,45 +377,95 @@ body {
+					text-decoration: none !important;
+				}
+
+				&:focus {
+					outline: 2px solid variables.$color-blue !important;
+					outline-offset: 2px !important;
+				}
 			}
 
 			&-code {
 				color: variables.$color-black !important;
 				display: block;
```

Stylelint (17.7.0)

[error] Duplicate property "display" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)